lib/elixir/test/elixir/port_test.exs

```diff
@@ -18,4 +18,17 @@ defmodule PortTest do
     assert Port.info(port, :registered_name) == nil
     assert Port.info(port) == nil
   end
+
+  test "monitoring functions are inlined by the compiler" do
```

Comment:
I believe we don't test inlining.

Reply:
Well, [you do](https://github.com/elixir-lang/elixir/blob/master/lib/elixir/test/elixir/process_test.exs#L27). But if you think it's better to test `:erlang.monitor` and `:erlang.demonitor` functionality here, I can do it also. :)

Reply:
Hmm, we do a lot of inlining and this is perhaps the only test we have so far. 🤔

Reply:
OK! I'll test the functionality then!